apps/src/StudioApp.js

```diff
@@ -1159,6 +1168,24 @@ StudioApp.prototype.handleEditCode_ = function (options) {
       enableLiveAutocompletion: true
     });
 
+    this.dropletTooltipManager = new DropletTooltipManager();
+    this.dropletTooltipManager.registerBlocksFromPalette(fullDropletPalette);
+
+    var installTooltips = function () {
+      this.dropletTooltipManager.installTooltipsOnVisibleToolboxBlocks();
+    }.bind(this);
+
+    this.editor.on('changepalette', installTooltips);
+
+    this.editor.on('toggledone', function () {
+      if (!$('.droplet-hover-div').hasClass('tooltipstered')) {
+        installTooltips();
+      }
+    });
+
+    // TODO(bjordan): is there a better hook for post-initial-load?
+    window.setTimeout(installTooltips, 500);
```

Comment:
It does seem like there should be some sort of this.editor.on('load') or something, but I don't currently know droplet well enough to be able to say for sure.